docroot/modules/custom/sitenow_intranet/sitenow_intranet.module

```diff
@@ -10,97 +10,105 @@
  */
 
 use Drupal\Core\Url;
+use Drupal\sitenow_intranet\IntranetHelper;
 
 /**
- * Implements hook_restrict_ip_access_denied_page_alter().
+ * Implements hook_preprocess_block().
  */
-function sitenow_intranet_restrict_ip_access_denied_page_alter(&$page) {
-  $config = \Drupal::config('sitenow_intranet.settings');
-  \Drupal::service('renderer')->addCacheableDependency($page, $config);
```

Comment:
@pyrello do you recall why this was removed/not moved to the preprocess page?

Reply:
Mostly because I wasn't sure if it was necessary. I believe there is a todo message about checking what we need to do for caching.

Reply:
I am a bit confused about this. DEV is showing private uncachable on the requests and unless restrict_ip is doing something else, this was the only thing I saw related to caching that was changed in this PR. That doc says:

`Otherwise, the passed in object must be assumed to be uncacheable, so max-age 0 is set.`

My guess would be to add this back in and go from there. 🤷

Reply:
Rick had done this instead of changing the performance cache setting from 1 month for the split as seen on studentlife.connect PROD

Reply:
I think restrict IP must have been doing something else. Just from looking at this, my guess would be that its effect would be to clear the cache when the config settings are changed.

Reply:
OTOH I don't see any instances in `restrict_ip` where any class or method with the word 'cache' in it.

Reply:
>would be to clear the cache when the config settings are changed
which we also want correct?

in restrict_ip doing a string search, only the readme mentions `cache`. no results for invalidate or other cache terminology.

Reply:
This was re-added in a new place.